.storybook/webpack.config.js

```diff
@@ -21,9 +21,9 @@ module.exports = {
         use: ['babel-loader', 'ts-loader'],
       },
       {
-        test: /\.svg$/,
-        exclude: [paths.publicFiles],
-        loader: 'svg-sprite-loader',
+       test: /\.svg$/i,
+        issuer: /\.[jt]sx?$/,
+        use: ['@svgr/webpack'],
```

Comment:
<img width="664" alt="Screenshot 2023-02-09 at 11 58 51" src="https://user-images.githubusercontent.com/25006856/217780198-4141973c-b888-4350-80e0-878919b9431b.png">
@vozbrann could check if it can be helpful for us?

Reply:
Not really in our case. Will require types refactoring.
The only usage of imported icon:
<img width="179" alt="image" src="https://user-images.githubusercontent.com/39827437/217800282-24930d06-9392-4241-bb9c-c5d82537b521.png">